common/changes/@microsoft/api-documenter/ianc-build-with-heft_2020-08-11-05-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "type": "none",
```

Comment:
@iclanton Switching to a completely different toolchain is a somewhat interesting change.  I wonder if we should publish a patch version of all these projects, so if there are any regressions, they have a version and changelog rather than being mixed up with other unrelated work.  Also it's free press for Heft. :-)

Reply:
That's a good idea.